docs/examples/use_cases/tensorflow/efficientdet/dataset/create_coco_tfrecord.py

```diff
@@ -325,11 +325,9 @@ def _get_caption_annotation(image_id):
 
 def main(_):
     assert FLAGS.image_dir, "`image_dir` missing."
-    assert (
-        FLAGS.image_info_file
-        or FLAGS.object_annotations_file
-        or FLAGS.caption_annotations_file
-    ), ("All annotation files are " "missing.")
+    assert FLAGS.image_info_file or \
+           FLAGS.object_annotations_file or \
+           FLAGS.caption_annotations_file, ("All annotation files are " "missing.")
```

Comment:
Small nitpick, looks like it is some leftover.
```suggestion
           FLAGS.caption_annotations_file, "All annotation files are missing."
```

Reply:
Will do in followup.